src/Reports/Endpoints/TotalIncome.php

```diff
@@ -144,8 +154,22 @@ public function get_trend( $start, $end, $income ) {
 
 		$prevEnd = clone $start;
 
-		$prevIncome    = $this->get_income( $prevStart->format( 'Y-m-d' ), $prevEnd->format( 'Y-m-d' ) );
-		$currentIncome = $this->get_income( $start->format( 'Y-m-d' ), $end->format( 'Y-m-d' ) );
+		// $prevIncome    = $this->get_income( $prevStart->format( 'Y-m-d' ), $prevEnd->format( 'Y-m-d' ) );
```

Comment:
@kjohnson Can explain how you are calculating trends with the same data? I can not see trends in `Total Revenue`

![image](https://user-images.githubusercontent.com/1784821/108051910-7450cd80-7071-11eb-9681-dcaf714d368f.png)